tools/project_manager/add_files.php

```diff
@@ -378,11 +382,12 @@ function analyze()
         // --------------------------------
 
         // Find out how long the 'image' field is.
-        $res = mysqli_query(DPDatabase::get_connection(), "
+        $sql = sprintf("
             SELECT image
-            FROM $this->projectid
```

Comment:
We can't escape table names like this. Instead, we should call `validate_projectID()` before we build the SQL to validate it, eg:
```
validate_projectID($this->projectid);
$sql = "
    SELECT image
    FROM $this->projectid
";
```